updated product strategy (#11812)
PostHog/posthog.com: contents/handbook/why-does-posthog-exist.md

## Our mission

> Equip every developer to build successful products.

Reviewer: Is this still the mission? 🙈 We don't talk about it much, which kind of makes me think it isn't resonating as much as it once did. Maybe we need to either rethink it, or make a purpose out of talking about it more.

## A few words

> As you know, the initial products of PostHog are a bunch of different SaaS tools, which work together better than everyone else’s. However, what most people may not know, is that our long term plan is to automate how entire businesses operate built around the unit they operate on - their customer data.

Reviewer: This is the very first chapter, so "As you know" makes me wonder if I've missed something.

```suggestion
On the surface, PostHog's products are a bunch of different SaaS tools, which work together better than anyone else’s. However, the secret to our success is our long term plan: automate how entire businesses operate by building around the unit they operate on - their customer data.
```

I don't like the "secret to our success" language but something along those lines..

Author: agree - think the secret thing is slightly the wrong ligno, i'll incorporate this then remove that

> Critical to making this happen is to deliver each tool without compromises. The way to achieve this is very simple - shipping every product that teams use, from day one when there are just two technical cofounders. This is because if we get in with these developers, we’re upstream of every need they’ll have. The end result is that we can offer all the products in one place, with no need to charge as much as everyone else for each one, and most importantly with perfect quality customer data. An actual perfect 360 customer view.

Reviewer: ```suggestion
Critical to making this happen is to deliver each tool without compromise. The way to achieve this is very simple - shipping every product that teams use, from day one when there are just two technical cofounders. This is because if we get in with these developers, we’re upstream of every need they’ll have. The end result is that we can offer all the products in one place, with no need to charge as much as everyone else for each one, and most importantly with perfect quality customer data. An actual perfect 360 customer view.
```

"Shipping each tool without compromise" doesn't really lead to the rest of the paragraph - shipping all the tools in one. What's missing is either how "no compromises means they can effectively use all the tools" or "no compromises means we build literally anything they can think of".... and I'm not sure what you meant here.

Author: it's a bit pithy, but i mean better _and_ cheaper, i'll adjust this language

Even so, it’s right to question whether this actually does any good for the world. Are we really in need of better software? And beyond those in the most competitive industries, will perfect customer data actually make a difference to an everyday person’s ability to build a successful business?

> No and not really. However, that misses the point, unless you understand the secret master plan alluded to above. The strategy of PostHog is to get early stage software teams to use every single customer related product inside our platform, where we can easily generate a perfect customer data record. This means we can learn to configure and automate how all of these tools work. We can simplify the experience of running a business.

Reviewer: ```suggestion
No and not really. However, that misses the point, unless you understand the secret master plan alluded to below. The strategy of PostHog is to get early stage software teams to use every single customer related product inside our platform, where we can easily generate a perfect customer data record. This means we can learn to configure and automate how all of these tools work. We can simplify the experience of running a business.
```

It's at the bottom - did you mean below?

> Then with each successive product, we can reach a wider and wider market and further simplify the experience of running a business until anyone can do it.

Reviewer: But how does _this_ do any good for the world? Is it because successful businesses make the world a better place? Because business owners are happier people? Because more businesses means more options for consumers? It could be any of those..... but the way it's written it just feels like this leaves a gap.

Author: i think all of the above - like the world is more productive so living standards are higher, it's more accessible to create a company, the world is more inspiring and less "set in stone", so will leave intentionally vague!

I’d like to address a repeated argument against shipping multiple products, let alone all of them.

### You can build more than a product or two

The world of tech is changing fast.

First, there’s just more money at better valuations than when this advice was created. Software companies, us included, have more leverage than ever before - with capital deployed well, we can do more than our predecessors.

Second, software development is far quicker.

Third, we’ve managed this very well so far. 

## Our strategy

### 1. Be the source of truth for customer and product data

Building a successful product is hard; doing so when you don't understand your customers is even harder. It's wild that no one has already provided a complete customer record. This has happened because the entire industry has focused on integration instead of consolidation.

> Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data in them. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.

Reviewer: Is "hundreds" an exaggeration for the bottom 99% of businesses? If so we should probably remove.

Reviewer: ```suggestion
Traditionally, as companies scale, their data warehouse becomes the source of truth, and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data they collect, simply because they are misused and divorced from the source of truth. Every company winds up with a huge mess of data spaghetti, with their business logic still spread across dozens or hundreds of tools.
```

I feel like we need an explanation of why product analytics is still relevant (it is!!!)

Author: disagree on hundreds, we'll wind up upmarket where this is the case - that part will eventually matter to us

> We provide customer infrastructure - by providing _every_ customer data related tool in one place, we can:

Reviewer: This should be hyphenated, but that's awkward, so rearranging

```suggestion
We provide customer infrastructure - by providing _every_ tool related to customer data in one place, we can:
```

- Enhance the utility of all the tools when used together
- Increase trust in data by eliminating complex data stacks
- Automate everything better than anyone else can, by using AI across this wider data context
- Continue to provide all the tools companies need as they grow

### 2. Provide every tool needed to build successful products

> We aim to offer every tool software teams need to find, sell to, get users, understand who they are, support them and improve their product experience.

Reviewer: ```suggestion
We aim to offer every tool software teams need to find, acquire, understand, support and sell to users so they can improve their product experience.
```

Author: this is a way nicer way to put it

> By doing this, we wind up with the perfect customer data record. No integrations needed.

Reviewer: Then why do we offer a CDP? should we explain that here?

Author: i think this is a bit edge casey (it's a necessary evil for now as we'll always have some gaps in tool coverage ie ad stuff) and too much depth for this page

> We can then get our AI to work across all of them together, whilst making every individual tool cheaper than the rest of the market - since we provide so many we can charge less.

Reviewer: I think it's more about - shared infra means lower individual product costs - there aren't 10 platforms ingesting your data and then doing whatever with it, just one.

### 3. Get in first

> Since developers exist first in a startup, by getting in with them early, we are upstream naturally of every single tool they would have otherwise used. Although anyone can pick up our products, this means we can best deliver customer infrastructure to early stage companies so should focus there by default.

Reviewer: ```suggestion
Since developers exist first in a startup, by getting in with them early, we are naturally upstream of every other tool they might have considered using. Although anyone can pick up our products (and lots of mature companies certainly do), this means we can best deliver customer infrastructure to early stage companies, and so should focus there by default.
```
